### PR TITLE
Allow configuration of autoscaler parameters in knative

### PR DESCRIPTION
**Pull request by timcharper**
**What type of PR is this?**
Feature

**What this PR does/ why we need it**:
When knative is deployed via flux, the generated autoscaler configmap is not configurable because flux will overwrite the configmap on next reconciliation.

**Which issue(s) this PR fixes**
no issue

**Special notes for your reviewer**:
I have tested this chart by installing it with kommander, and have confirmed that the parameters are configurable.

**Does this PR introduce a user-facing change?**:
yes

```release-note
Knative serving autoscaler parameters are now configurable via helm vars.
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [x] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [x] The relevant subset of integration tests pass locally.
* [x] The core changes are covered by tests.
* [x] The documentation is updated where needed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,6 +29,9 @@ GIT_REMOTE_URL ?= $(shell git remote get-url ${GIT_REMOTE_NAME})
 # - git@github.com:mesosphere/charts.git
 GITHUB_USER := $(shell git remote get-url ${GIT_REMOTE_NAME} | sed -E 's|.*github.com[/:]([^/]+)/charts.*|\1|')
 
+
+REPO_URL := https://$(GITHUB_USER).github.io/charts
```
Comment (timcharper) [inline Makefile:33]: Maybe this would be better named REPO_BASE_URL ?

**Comment by timcharper** [comment]
@gracedo this PR does not update the component version. Should I add a `-1` to the end of the chart version, to confer a packaging change only?

**Comment by gracedo** [comment]
@timcharper sorry I completely missed this - can we bump the chart version? This looks like it's adding functionality to to the helm chart modify values, which justifies a chart bump

**Review by lukeogg — APPROVED** [review]
LGTM!

**Review by andresmascl — APPROVED** [review]
Looks good to me too!

**Review by gracedo — CHANGES_REQUESTED** [review]
needs a chart version bump, please

**Comment by timcharper** [comment]
> needs a chart version bump, please

How to we bump the chart version? `-1` suffix or something?

**Comment by gracedo** [comment]
https://github.com/mesosphere/charts/blob/tharper/knative-activator-capacity/staging/knative/Chart.yaml#L3
This should be bumped to maybe 0.4.0 since it's adding new functionality

**Comment by juliangieseke** [comment]
I think https://github.com/mesosphere/charts/blob/tharper/knative-activator-capacity/staging/knative/charts/serving/Chart.yaml#L3 needs a bump too.

**Review by gracedo — APPROVED** [review]

**Pull request merged**